dev-libs/libtypec/metadata.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+  <longdescription>
+    “libtypec” is aimed to provide a generic interface abstracting all
+    platform complexity for user space to develop tools for efficient
+    USB-C port management. The library can also enable development of
+    diagnostic and debug tools to debug system issues around USB-C/USB
+    PD topology.
+  </longdescription>
+  <maintainer type="person">
+    <name>Adrian Ratiu</name>
+    <email>adrian.ratiu@collabora.com</email>
+  </maintainer>
+  <maintainer type="person">
```

Comment:
`<maintainer type="person" proxied="yes">`, of if he's not expected to contribute to `::gentoo` you could just list him as the upstream maintainer within `<upstream>...</upstream>` -- see `app-accessibility/espeak-ng/metadata.xml` for example.